gdnative-core/src/core_types/vector3.rs

```diff
@@ -15,6 +15,16 @@ pub enum Axis {
 pub trait Vector3Godot {
     /// Internal API for converting to `sys` representation. Makes it possible to remove
     /// `transmute`s elsewhere.
+
+    /// Returns the normalized vector pointing from this vector to `other`.
+    fn direction_to(self, other: Vector3) -> Vector3;
+    /// Returns the distance to `other`.
+    fn distance_to(self, other: Vector3) -> f32;
+    /// Returns the squared distance to `other`.
+    /// This method runs faster than distance_to, so prefer it if you need to compare vectors or
```

Comment:
An empty doc-comment line should be inserted between the summary and detailed descriptions.